buzsaki_lab_to_nwb/original-code/convert_yuta.py

```diff
@@ -224,30 +225,34 @@ def yuta2nwb(session_path='D:/BuzsakiData/SenzaiY/YutaMouse41/YutaMouse41-150903
     print('setting up electrodes...', end='', flush=True)
     hilus_csv_path = os.path.join(fpath_base, 'early_session_hilus_chans.csv')
     lfp_channel = get_reference_elec(subject_xls, hilus_csv_path, session_start_time, session_id, b=b)
-    
-    
+
     custom_column = [{'name': 'theta_reference',
                       'description': 'this electrode was used to calculate LFP canonical bands',
                       'data': all_shank_channels == lfp_channel}]
     ns.write_electrode_table(nwbfile, session_path, custom_columns=custom_column, max_shanks=max_shanks)
-    
-    print('reading raw electrode data...', end='', flush = True)
+
+    print('reading raw electrode data...', end='', flush=True)
+    xml_filepath = os.path.join(session_path, session_id + '.xml')
+    xml_root = et.parse(xml_filepath).getroot()
+    acq_saampling_frequency = float(xml_root.find('acquisitionSystem').find('samplingRate').text)
```

Comment:
shouldn't this be in the if statement below?

Reply:
We can indeed move it inside the first stub call to remove unnecessary access outside of that statement. Fixed.